e2e/testdata/TestCreatesInsecureCluster/creates_3-node_insecure_cluster.golden

```diff
@@ -15,7 +15,6 @@ spec:
 apiVersion: v1
 kind: Pod
 metadata:
-  creationTimestamp: null
```

Comment:
I don't see anything that would obviously result in these lines being removed?

Reply:
It is how the test is failing.  It is a difference between GKE and Kind :(